ownCloud/Theming/NSObject+ThemeApplication.swift

```diff
@@ -80,7 +80,7 @@ extension NSObject {
 			let navigationController : UINavigationController = (self as? UINavigationController)!
 
 			navigationController.navigationBar.applyThemeCollection(collection, itemStyle: itemStyle)
-			navigationController.view.backgroundColor = collection.tableBackgroundColor
+			//navigationController.view.backgroundColor = collection.tableBackgroundColor
```

Comment:
I've left this line commented because when you launch a ThemeNavigationController with a RootViewController that has some transparency or blur view, this navigation controller background color interferes with the background color of your previous view you only see this color or this color with a blur. I've checked the app with the different themes we already support and I didn't notice any change. 

Is there another better solution taking advantage of our Theme support mechanism?

Reply:
Good point! Did you notice any regressions or visual glitches anywhere by leaving this line out?